src/groovy/au/org/emii/portal/DownloadReport.groovy

```diff
@@ -6,9 +6,12 @@ import java.text.DateFormat
 
 class DownloadReport {
 
+    static final def REPORT_TEMP_FILE_SUFFIX = "download_report"
+    static final def ZIP_FILE_NAME = "download_report.txt"
```

Comment:
this variable name seems confusing to me. Once you understand the code it's clear it means "filename to give the report file in zip" but initially it seems like it must mean "name of the zip file"